course/templates/course/course_base.html

```diff
@@ -79,26 +79,38 @@ <h4>{% trans "Calendar" %}</h4>
             {% language user_language %}
             <div lang={{user_language}}>
                 <div class="alert alert-default">
-                    {% trans "You are browsing this course in different language than set in your preferences." %}
+                    {% trans "You are browsing this course in a different language than is set in your preferences." %}
                     {% if user_course_data %}
                         <div class="row">
                             <div class="col-lg-4">
                                 <form action="{{ url_without_language }}">
-                                    <button class="btn btn-default btn-block" type="submit">{% blocktrans with lang=user_language|language_name_translated %}browse the course in {{ lang }}{% endblocktrans %}</button>
+                                    <button type="submit" class="aplus-button--secondary aplus-button--md btn-block">
+                                        {% blocktrans with lang=user_language|language_name_translated %}
```

Comment:
Can you add `trimmed` here. It will make the translation strings nicer. (`{% blocktrans trimmed with lang=user_language|language_name_translated %}`)